src/octopeer-github/content/ElementEventBinding/ElementEventBindingData.ts

```diff
@@ -37,11 +37,45 @@ const elementEventBindingDataList: ElementEventBindingData[] = [
         name: "mouseleave",
     },
     {
-        eventID: EventID.SCROLL_INTO_VIEW,
-        name: "scrollintoview",
-    },
-    {
-        eventID: EventID.SCROLL_OUT_OF_VIEW,
-        name: "scrolloutofview",
+        addDOMEvent: (elementSelectionBehaviour: ElementSelectionBehaviour) => {
+            let wasInScope = false;
+            let handleScrollEvent = (esb: ElementSelectionBehaviour, eventObject: JQueryEventObject) => {
+                let isBetween = (leftBound: number, rightBound: number, value: number) => {
```

Comment:
As just discussed in person with @Mathias, this was a way to show where
functions could be moved to an auxilary class, as is done in commit 56a8a1b
<https://github.com/thervh70/ContextProject_RDD/commit/56a8a1bda64f92fd43a9639276886cf01e0a89fe>
...

Met vriendelijke groet,
Youri Arkesteijn

On Thu, Jun 16, 2016 at 1:41 PM, MathiasMeuleman <notifications@github.com>
wrote:

> In
> src/octopeer-github/content/ElementEventBinding/ElementEventBindingData.ts
> <https://github.com/thervh70/ContextProject_RDD/pull/161#discussion_r67329928>
> :
>
> > @@ -37,11 +37,45 @@ const elementEventBindingDataList: ElementEventBindingData[] = [
> >          name: "mouseleave",
> >      },
> >      {
> > -        eventID: EventID.SCROLL_INTO_VIEW,
> > -        name: "scrollintoview",
> > -    },
> > -    {
> > -        eventID: EventID.SCROLL_OUT_OF_VIEW,
> > -        name: "scrolloutofview",
> > +        addDOMEvent: (elementSelectionBehaviour: ElementSelectionBehaviour) => {
> > +            let wasInScope = false;
> > +            let handleScrollEvent = (esb: ElementSelectionBehaviour, eventObject: JQueryEventObject) => {
> > +                let isBetween = (leftBound: number, rightBound: number, value: number) => {
>
> Could you first read others comments before commenting? Would reduce the
> amount of comments considerably
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/thervh70/ContextProject_RDD/pull/161/files/e639bcb6823ac78360fa06836329eee24aa897d8#r67329928>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAdBI4Stg3P8IRhurzerJa7j9wxvOiIqks5qMTZegaJpZM4I2QY1>
> .
>